### DISCUSSION TITLE
✅ macOS bulk delete server entries syncing to iCloud

### DISCUSSION THREAD

**Discussion by ghost** [isontheline/pro.webssh.net#1080] [Q&A] [answered]
Unable to select multiple remote machines at the same time and not looking forward to going through my list of 200+ machines one-at-a-time deleting them. 

Is this possible?

**Comment by isontheline** [comment, marked as answer]
Hello @pc-support65 👋

Right now there is no ability to do that in GUI but you can remove in batch by using the command line :
https://webssh.net/documentation/help/howtos/mashREPL/servedit/#examples-delete-connections

Be careful as it can remove all your data.
When you are confident with the filter => use `--apply`
